### ISSUE TITLE
which site covariate for model matrix?

### ISSUE THREAD

**Issue by klg-2016** [kaitlyngaynor/gorongosa-mesocarnivores#126]
https://github.com/kaitlyngaynor/gorongosa-mesocarnivores/blob/d43955d197f0e8d86a5a75bd03e9bfab1e57dd6e/occupancy-mmpp-master/application/02_fit_model_meso.R#L531

this is another instance of my not having a solid enough understanding of how they used their site covariates (Hunting in this instance) to know which GNP covariate makes the most sense (I arbitrarily chose one of the lion variables here). I also just realized I need to check whether their covariates are scaled/whether the GNP ones should be

**Comment by kaitlyngaynor** [comment]
Yes, good to scale covariates prior to modeling, as a general rule!

As I mentioned in the other issue, I'd have a closer look at the paper (perhaps both papers) to try to get to the bottom of this. I would advise that you think about this process not as "swapping out" their covariates for yours, but thinking through the hypotheses you want to test, and how they relate to the hypotheses that Kellner et al. tested. Then think about how to adapt the model to test the hypotheses you are interested in!
